apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -19,6 +19,7 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 
 		this.draggable = props.draggable || true;
 		this.cardType = props.cardType;
+		if (props.completionDate) this.completionDate = props.completionDate;
```

Comment:
see also note in card.types about ICardProprieties - the prop should be defined part of ITaskCardProps above